components/external-services-mock/README.md

```diff
@@ -0,0 +1,19 @@
+# External services mock
+
+## Overview
+
+External services mock is a test component that implement external APIs to which compass sends requests.
```

Comment:
Suggestion: `That mocks external APIs for test purposes.`

List here which mocks we have currently